SignalServiceKit/tests/Contacts/TSContactThreadTest.m

```diff
@@ -19,7 +19,6 @@ @implementation TSContactThreadTest
 
 - (void)setUp
 {
```

Comment:
This doesn't call super. Can you check the others to make sure they all do?

Maybe we can take advantage of `__attribute__((objc_requires_super));`

Also, this method, and presumably others are doing things to ensure no existing state lingers. That should be unnecessary now, right?

In this example, we could get rid of `[OWSRecipientIdentity removeAllObjectsInCollection]`?

Reply:
> This doesn't call super. Can you check the others to make sure they all do?

Done.

> Maybe we can take advantage of __attribute__((objc_requires_super));

I've added this to the card.

> Also, this method, and presumably others are doing things to ensure no existing state lingers. That should be unnecessary now, right?

Yup, that's right.  Since we're re-doing this work in small steps, I'm trying to first work towards a) getting everything building b) mechanical changes.

> In this example, we could get rid of [OWSRecipientIdentity removeAllObjectsInCollection]?

Right, there's a lot of that kind of "teardown" that will no longer be necessary and which I'll eventually remove.